spec/features/user_can_sign_out_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+class PostsTests < ActionDispatch::IntegrationTest
+  include Devise::Test::IntegrationHelpers
+end
+
+feature 'can sign out of an account', :devise do
+  scenario 'can sign out' do
+    user = FactoryGirl.create(:user)
+    visit '/users/sign_in'
+
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+
+    click_button 'Log in'
+    expect(page).to have_content I18n.t 'devise.sessions.signed_in'
```

Comment:
Make sure the expectations are on their own section (add a line break above the first one). It makes it much more explicit. While the line breaks/ organization decisions for the rest of the test are more flexible, that is pretty much a rule by most people's standards